pyuvdata/utils.py

```diff
@@ -746,26 +747,46 @@ def antnums_to_baseline(ant1, ant2, Nants_telescope, attempt256=False):
         return np.int64(baseline)
 
 
-def get_baseline_redundancies(baseline_inds, baseline_vecs, tol=1.0):
+def get_baseline_redundancies(baseline_inds, baseline_vecs, tol=1.0, with_conjugates=False):
     """
     Return redundant baseline groups, and lists of corresponding lengths and vectors
 
     Args:
         baseline_inds = Baseline indices.  (Nbls,), (int)
         baseline_vecs = Baseline vectors in meters. (Nbls, 3), (float)
         tol  = Absolute tolerance of redundancy, in meters. (float)
+        with_conjugates = Include baselines that are redundant when flipped.
 
     Returns:
         baseline_groups: list of arrays of redundant baseline indices
         vec_bin_centers: List of vectors describing redundant group centers
         lengths: List of redundant group baseline lengths in meters
+        baseline_ind_conj: List of baselines that are redundant when reversed
 
     """
     Nbls = baseline_inds.shape[0]
 
     if not baseline_vecs.shape == (Nbls, 3):
         raise ValueError("Baseline vectors must be shape (Nbls, 3)")
 
+    baseline_vecs = copy.copy(baseline_vecs)              # Protect the vectors passed in.
+    baseline_vecs = np.around(baseline_vecs, decimals=4)  # Round to nearest 0.1mm, to avoid floating errors from unphasing.
```

Comment:
I'm a little uncomfortable with hardcoding this, but I understand the problem you were trying to fix. I guess the good news is it's smaller than the tolerance on the uvws (which I believe is 1mm).

Reply:
That seems to work. It looks like unphasing with `use_ant_pos` assigns uvws as pos(ant2) - pos(ant1), while `get_antenna_redundancies` uses the opposite convention. I can switch that around, but then I'll get different uvws from what's already in the file. I'll keep at it.